admin_guide/cluster_management.rst

```diff
@@ -410,7 +410,6 @@ So far the information in this view doesn't give us anything we couldn't already
 
 ::
 
-  .
```

Comment:
The reason this dot is here is to set the indentation level of the code block. If the dot isn't present, Sphinx will consider the first line to be the one starting with `   sum` and will count three spaces of indentation. The horizontal line of dashes is less indented, however, and would end the code block.

The dot thing is a hack, no doubt about that. I just don't know a better way. Maybe you can think of one?